src/Objects/Mount.vala

```diff
@@ -1,6 +1,6 @@
 // -*- Mode: vala; indent-tabs-mode: nil; tab-width: 4 -*-
 /*-
- * Copyright (c) 2016-2018 elementary LLC. (https://elementary.io)
```

Comment:
but why

Reply:
Because this was written in 2018, and incorrectly had 2016 in there before